Bigtable/src/ChunkFormatter.php

```diff
@@ -198,57 +242,56 @@ private function commit()
     }
 
     /**
-     * Moves to next state in processing.
+     * Moves to the next state in processing.
      *
-     * @param \Google\Cloud\Bigtable\V2\ReadRowsResponse\CellChunk $chunk in process.
+     * @param CellChunk $chunk The chunk to analyze.
      * @return void
      */
-    private function moveToNextState($chunk)
+    private function moveToNextState(CellChunk $chunk)
     {
-        if ($chunk->getValueSize() > 0) {
-            $this->state = $this->RowStateEnum['CELL_IN_PROGRESS'];
-        } else {
-            $this->state = $this->RowStateEnum['ROW_IN_PROGRESS'];
-        }
+        $this->state = $chunk->getValueSize() > 0
+            ? self::$rowStateEnum['CELL_IN_PROGRESS']
+            : self::$rowStateEnum['ROW_IN_PROGRESS'];
     }
 
     /**
-     * Process chunk when in NEW_ROW state.
+     * Processes a chunk when in the NEW_ROW state.
      *
-     * @param \Google\Cloud\Bigtable\V2\ReadRowsResponse\CellChunk $chunk chunk to process.
+     * @param CellChunk $chunk The chunk to process.
      * @return void
+     * @throws BigtableDataOperationException
      */
-    private function newRow($chunk)
+    private function newRow(CellChunk $chunk)
     {
-        $newRowkey = $chunk->getRowKey();
         $this->validateNewRow($chunk);
-        $this->rowKey = $newRowkey;
+        $this->rowKey = $chunk->getRowKey();
         $familyName = $chunk->getFamilyName()->getValue();
         $qualifierName = $chunk->getQualifier()->getValue();
-        $labels = ($chunk->getLabels()->getIterator()->valid())?
-            implode(iterator_to_array($chunk->getLabels()->getIterator())):'';
-        $timestamp = $chunk->getTimestampMicros();
+        $labels = ($chunk->getLabels()->getIterator()->valid())
```

Comment:
Is there a reason we don't want to return the labels as an array?

Reply:
We need to talk with @sduskis I think currently server supports just one label.